app/controllers/candidate_interface/sign_in_controller.rb

```diff
@@ -2,6 +2,7 @@ module CandidateInterface
   class SignInController < CandidateInterfaceController
     skip_before_action :authenticate_candidate!
     before_action :redirect_to_application_if_signed_in, except: %i[confirm_authentication authenticate]
+    before_action :redirect_if_one_login_enabled
```

Comment:
Not a blocker but it would be nice to be a bit more explicit here about where the user is being redirected to, like the one on line 4. 

`redirect_to_sign_in_if_one_login_enabled` or something along those lines.